### ISSUE TITLE
The library thinks that when we are on a browser cordova will be undefined.

### ISSUE THREAD

**Issue by deliverymanager** [sahat/satellizer#977]
I think the reason that the library does not work on a browser with IONIC / cordova is the fact that it assumes that when we are on a browser this.$window['cordova'] will be undefined.

But actually this is not the case in an ionic project.
On a browser platform, after we build and upload the application on a server cordova exists!
So, if I am right and please correct me if I am wrong, in order to be able to use the library, you will need to replace in every if else condition (this.$window['cordova'] && this.$window['cordova'].platformId === "browser) in order to be able to work on a browser platform.

**Comment by fsalata** [comment]
As far as I know cordova.js does not exists when the project runs on a browser, it gets auto included while deploying on a real device/emulator. If you take a look at the contents of cordova.js file while running the project on a browser you will see something like this:
// mocked cordova.js response to prevent 404 errors during development
I believe that is the reason this.$window['cordova'] is undefined.

**Comment by jacquesdev** [comment]
Yup cordova plugins do not work in the browser. Usually in our code the plugin gets excluded when window.cordova is not defined or when ionic.platform === 'browser'
